lib/workos/mfa.rb

```diff
@@ -135,30 +134,48 @@ def challenge_factor(
         params(
           authentication_challenge_id: T.nilable(String),
           code: T.nilable(String),
-        ).returns(WorkOS::VerifyFactor)
+        ).returns(WorkOS::VerifyChallenge)
       end
       def verify_factor(
         authentication_challenge_id: nil,
         code: nil
       )
+        warn '[DEPRECATION] `verify_factor` is deprecated.
+        Please use `verify_challenge` instead.'
```

Comment:
Should we keep this on the same line?

```suggestion
        warn '[DEPRECATION] `verify_factor` is deprecated. Please use `verify_challenge` instead.'
```